src/main/java/org/jabref/logic/util/io/FileUtil.java

```diff
@@ -279,6 +283,50 @@ public static String createFileNameFromPattern(BibDatabase database, BibEntry en
     }
 
     /**
+     * Determines filename provided by an entry in a database
+     *
+     * @param database        the database, where the entry is located
+     * @param entry           the entry to which the file should be linked to
+     * @param fileNamePattern the filename pattern
+     * @return a suggested fileName
+     */
+    public static String createFileNameFromPattern(BibDatabase database, BibEntry entry, String fileNamePattern) {
```

Comment:
We are trying to get rid of these static helper methods. Similar to my remark above, I would propose to create a `FileNamePattern` class, that accepts the file name pattern and directory pattern as constructor arguments and provides these two methods as instance methods.

Reply:
The 'static String createFileNameFromPattern(...)' method was given, and I was trying to change as little as possible, so opted to make a static ''